apiserver/src/types/customRequest.d.ts

```diff
@@ -6,7 +6,7 @@ declare global {
 	namespace Express {
 		interface Request {
 			database: Database;
-			auth: Auth;
+			authent: Auth;
```

Comment:
Just curious, "auth" was already used ?

Reply:
Was pretty sure to have once a bug about `auth`. Not reproductive today, commit will come to restore `auth`.